tests/integration/components/hyper-table-v2/manage-columns-test.ts

```diff
@@ -0,0 +1,248 @@
+        await render(hbs`<HyperTableV2::ManageColumns @handler={{this.handler}} />`);
+        await click('.upf-btn.upf-btn--primary');
+
+        document.querySelectorAll('.fields-list .field').forEach((element, index) => {
+          assert.equal(
+            element.firstElementChild?.getAttribute('data-control-name'),
+            `column_definition_toggle_checkbox_${COLUMN_DEFINITIONS[index].key}`
+          );
+        });
+
+        const clusterNames = document.querySelectorAll('.fields-list .cluster-name');
+
+        assert.equal(clusterNames[0].textContent?.trim(), 'instagram');
+        assert.equal(clusterNames[1].textContent?.trim(), 'youtube');
+        assert.expect(6);
+      });
+
+      test('it sets category as active when user select a category', async function (assert) {
+        await render(hbs`<HyperTableV2::ManageColumns @handler={{this.handler}} />`);
+        await click('.upf-btn.upf-btn--primary');
+        await click('[data-control-name="field_category_toggle_affiliation"]');
+
+        assert.dom('[data-control-name="field_category_toggle_affiliation"]').hasClass('field-category--active');
+        assert.dom('[data-control-name="field_category_toggle_all_fields"]').doesNotHaveClass('field-category--active');
+      });
+
+      test('it display column definitions of active category with clustering key', async function (assert) {
+        await render(hbs`<HyperTableV2::ManageColumns @handler={{this.handler}} />`);
+        await click('.upf-btn.upf-btn--primary');
+        await click('[data-control-name="field_category_toggle_influencer"]');
+
+        const columnDefinitionsVisible = document.querySelector('.fields-list')?.children || [];
+
+        assert.equal(columnDefinitionsVisible[0].textContent?.trim(), 'instagram');
+        assert.equal(
+          columnDefinitionsVisible[1].firstElementChild?.getAttribute('data-control-name'),
+          'column_definition_toggle_checkbox_foo'
+        );
+        assert.equal(columnDefinitionsVisible[2].textContent?.trim(), 'youtube');
+        assert.equal(
+          columnDefinitionsVisible[3].firstElementChild?.getAttribute('data-control-name'),
+          'column_definition_toggle_checkbox_bar'
+        );
+      });
+
+      test('it display column definitions of active category without clustering key', async function (assert) {
+        await render(hbs`<HyperTableV2::ManageColumns @handler={{this.handler}} />`);
+        await click('.upf-btn.upf-btn--primary');
+        await click('[data-control-name="field_category_toggle_affiliation"]');
+
+        const columnDefinitionsVisible = document.querySelector('.fields-list')?.children || [];
+
+        assert.equal(
+          columnDefinitionsVisible[0].firstElementChild?.getAttribute('data-control-name'),
+          'column_definition_toggle_checkbox_code'
+        );
+      });
+    });
+
+    module('when user manage column definition', function () {
```

Comment:
`when user manages ....`
line 157: `it removes the column in the table`
line 170: `it adds the column in the table`
line 227: `it displays the column in the table`